### PR TITLE
Add pre processed path to the input.

### PR DESCRIPTION
**Pull request by ashutosh-narkar**
Add path as a string array to the input to help with indexing.

Signed-off-by: Ashutosh Narkar <anarkar4387@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,9 @@
 This repository contains an extended version of OPA (**OPA-Istio**) that allows you to enforce OPA
 policies at the Istio Proxy layer.
 
+## Issue Management
+Use [OPA GitHub Issues](https://github.com/open-policy-agent/opa/issues) to request features or file bugs.
```
Comment (tsandall) [inline README.md]: Typically I'd do this in another commit :D

File: README.md [hunk 2]
```diff
@@ -209,6 +212,7 @@ The `input` value defined for your policy will resemble the JSON below:
 
 ```json
 {
+  "parsed_path": ["api", "v1", "products"],
```
Comment (tsandall) [inline README.md]: It's worth documenting this field a little bit more since everything else in the input comes from the standard Envoy ext_authz check request type. I'd explicitly call this field out and mention when/how to use it.

Comment (ashutosh-narkar) [reply README.md]: Done
